Dockerfile

```diff
@@ -62,21 +63,24 @@ RUN apt-get update && \
         /usr/share/man \
         /usr/share/doc \
         /usr/share/doc-base
-ADD hack/recurl.sh .
-RUN ./recurl.sh /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/${IMAGE_ARCH}/kubectl
+
+ADD hack/recurl.sh hack/image_arch.sh .
+RUN ./image_arch.sh && ./recurl.sh /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/${IMAGE_ARCH}/kubectl
 RUN ./recurl.sh /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64
 RUN rm recurl.sh
+
 COPY hack/ssh_known_hosts /etc/ssh/ssh_known_hosts
 COPY --from=builder /usr/local/bin/docker /usr/local/bin/
 
 ####################################################################################################
 
-FROM node:14.0.0 as argo-ui
+FROM node:14.0.0-alpine as argo-ui
```

Comment:
can you add some commentary about this change please?

Reply:
I can revert back the changes, that was just for me to reduce the time needed to download the image.
That also explains the timeout option for yarn ;)

Reply:
Is this faster? If so - please don't change it!

Reply:
I can test that, but I think if download speed is not an issue: No.

Reply:
Also reverted this